tiatoolbox/cli.py

```diff
@@ -2,9 +2,12 @@
 from tiatoolbox import __version__
 from tiatoolbox import dataloader
 from tiatoolbox import utils
+
 import sys
 import click
 import os
+import pathlib
+from PIL import Image
```

Comment:
Are we using opencv and pillow? Is there a reason for using both libraries?

Reply:
Also, can we fix the spacing in this file.

Reply:
> Are we using opencv and pillow? Is there a reason for using both libraries?

Pillow Image show opens the image in default system browser that's the intention here for command-line.

Reply:
> Also, can we fix the spacing in this file.

I don't see any issues with spacing can you point out?

Reply:
Go ahead if you are fine with it